setup.py

```diff
@@ -40,5 +40,7 @@ def read(fname):
         'cartopy': ['cartopy'],
         'geopandas': ['geopandas'],
         'osmnx': ['osmnx >= 0.16.1'],
+        "dev": ["pytest", "sphinx", "sphinx_rtd_theme", "geopandas"],
+        "tests": ["pytest", "sphinx", "sphinx_rtd_theme", "geopandas", "tox"],
```

Comment:
either "dev" or "tests" should be removed?

and "sphinx", "sphinx_rtd_theme" should be in `docs/requirements.txt` ?

Reply:
I agree:
```suggestion
        "tests": ["pytest", "tox"],
```
sphinx, sphinx_rtd_theme should go to docs/requirements.txt

Geopandas could be dropped, right?

Reply:
> 
> Geopandas could be dropped, right?

Actually, tox did not install all extra_requires. And, there are tests that need geopandas, as geopandas is also an optional requirement (the gistool modules need geopandas). So, either make tox use extra_requires for test, or leave geopandas there - what do you think?

Reply:
```suggestion
        "tests": ["pytest", "geopandas", "tox"],
```

Reply:
To me, it would make more sense to let tox install extras_require as well. Or move geopandas to the general requirements if it has become so central to DHNx. If we leave it like that and keep geopandas in extras_require, we could at least describe the reason in an inline comment.